response_operations_ui/views/messages.py

```diff
@@ -98,11 +99,22 @@ def view_conversation(thread_id):
                            form=form)
 
 
-@messages_bp.route('/', methods=['GET', 'POST'])
+@messages_bp.route('/', methods=['GET'])
 @login_required
 def view_select_survey():
+    if "messages_survey_selection" in session:
```

Comment:
I think this would read better:

```Python
try:
    selected_survey = session["messages_survey_selection"]
except KeyError:
    return redirect(url_for("messages_bp.select_survey"))
else:
    return redirect(url_for("messages_bp.view_selected_survey",
                             selected_survey=selected_survey))
```

Reply:
I think that may fail the pylint no-else-return rule. Without the else it would be:
```Python
try:
    selected_survey = session["messages_survey_selection"]
except KeyError:
    return redirect(url_for("messages_bp.select_survey"))

return redirect(url_for("messages_bp.view_selected_survey",
                         selected_survey=selected_survey))
```
vs
```Python
if "messages_survey_selection" in session:
    return redirect(url_for("messages_bp.view_selected_survey",
                             selected_survey=session["messages_survey_selection"]))
 
return redirect(url_for("messages_bp.select_survey"))
```
I'm torn as to which one reads better 🤷‍♂️